integration-tests/src/test/resources/bash-scripts/install-istio.sh

```diff
@@ -23,25 +26,23 @@ workdir=$2
 istiodir=${workdir}/istio-${version}
 echo "Installing Istio version [${version}] in location [${istiodir}]"
 
-helm repo add istio.io https://storage.googleapis.com/istio-release/releases/${version}/charts/
-
 kubectl delete namespace istio-system --ignore-not-found
-kubectl create namespace istio-system
+#kubectl create namespace istio-system
```

Comment:
The ns is deleted but creating the ns is ocmmented out. If it is created by instrall, can you please add a comment saying that?

Reply:
Added the comments 
>
istio installation will create the namespace 'istio-system
>